nextflow.config

```diff
@@ -157,6 +157,11 @@ params {
     // profile standardisation
     run_profile_standardisation             = false
     standardisation_taxpasta_format         = 'tsv'
+    taxpasta_taxonomy_dir                   = false
```

Comment:
```suggestion
    taxpasta_taxonomy_dir                   = null
```

I think? It's not a boolean, it's a path?

Reply:
Yes, it is a path.